installer/scripts/omsadmin.sh

```diff
@@ -70,6 +70,12 @@ OMSCLOUD_ID=""
 UUID=""
 USER_ID=`id -u`
 
+DEFAULT_SYSLOG_PORT=25224
+DEFAULT_MONITOR_AGENT_PORT=25324
```

Comment:
Is there any reason that we would ever try to claim more than 100 syslog ports, thus overflowing into the monitor agent ports? In this case, should the find_available_port() method simply skip past the monitor agent ports?